### PR TITLE
Update mui

### PR DESCRIPTION
**Pull request by GraceMattern**
- update mui + inherit theme
- delete signIn/signUp component files
- ran prettier

### CODE REVIEW

**Review by roccopolimen — COMMENTED** [review]
Just a suggestion if you want to change, otherwise LGTM

File: client/src/pages/Landing/index.tsx [hunk 1]
```diff
@@ -1,93 +1,188 @@
...
+                      }
+                    >
+                      Sign up
+                    </Button>
+                    <Button
+                      variant="outlined"
+                      color="primary"
+                      id="signin"
+                      onClick={() =>
+                        (window.location.href = "http://localhost:3000/signin")
```
Comment (mpolimen) [inline client/src/pages/Landing/index.tsx]: Use the useNavigate hook, "window.location.href" makes this no longer a single page application, which is a strict requirement

**Review by mpolimen — CHANGES_REQUESTED** [review]

File: client/src/pages/Landing/index.tsx [hunk 2]
```diff
@@ -1,93 +1,185 @@
...
+                >
+                  Login
+                </Button>
+                <Button
+                  id="github-button"
+                  color="inherit"
+                  onClick={() =>
+                    (window.location.href =
+                      "https://github.com/roccopolimen/nextround")
+                  }
```
Comment (GraceMattern) [inline client/src/pages/Landing/index.tsx:53]: Should I remove this then because this is a direct redirect, not a single page change

**Comment by GraceMattern** [comment]
> Just a suggestion if you want to change, otherwise LGTM

What suggestion are you referring to?

File: client/src/pages/Landing/index.tsx (hunk 2 above)
Comment (mpolimen) [reply client/src/pages/Landing/index.tsx:53]: This is good, I think

**Review by mpolimen — APPROVED** [review]

**Pull request merged**